app/src/main/res/layout/conceptcard_fragment.xml

```diff
@@ -0,0 +1,27 @@
+<androidx.constraintlayout.widget.ConstraintLayout android:layout_height="match_parent"
+  android:layout_width="match_parent"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:background="@color/colorIvory">
+  <androidx.appcompat.widget.Toolbar
+    android:id="@+id/toolbar"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/colorConceptToolbar"
+    app:layout_constraintTop_toTopOf="parent"
+    app:layout_constraintLeft_toLeftOf="parent"
+    app:layout_constraintRight_toRightOf="parent" />
+  <ScrollView
+    android:layout_width="match_parent"
+    android:layout_height="0dp"
+    app:layout_constraintTop_toBottomOf="@id/toolbar"
```

Comment:
Nit: as a general practice, I suggest always referencing other view IDs in constraint layout as though you haven't seen the ID yet, e.g.:

``"@+id/toolbar"``

This makes it much easier to move things around, and define things out of order. Android will ensure all occurrences of the toolbar ID is reused.